web/client/src/util/apiWrapper.js

```diff
@@ -5,11 +5,20 @@ const BASE_URL =
     ? "https://yousefa-step-2020.uc.r.appspot.com/api"
```

Comment:
Maybe add a TODO comment here for the long-term plan of the production endpoint as a reminder to update this. Also, maybe ask the intern web-dev consultants on the best way to host this app post-internship. Thanks!

Reply:
Will do, no problem!